argocd/minio/Chart.yaml

```diff
@@ -2,6 +2,8 @@
 apiVersion: "v2"
 name: "minio"
 version: "0"
+description: |
+  This chart installs MinIO, a High Performance Object Storage.
```

Comment:
Maybe add a note that this app is used on environments that don't provide an Object Storage solution.

Reply:
updated text following suggestsions